### PR TITLE
Added Password Generator

### LINKED ISSUE #45: Password Generator
### Issue Type:
Feature Request

### Describe the feature
I would like to add Password Generator project.

### PR DESCRIPTION
**Pull request by jigar-sable**
# Description

Added Password Generator through which user can generate any type of password including letters, numbers and special chars, etc. Password length is user's choice. User can copy it on a single click.

Fixes #45 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
Deployed Link : [Visit](https://jigar-sable.github.io/Amazing-Js-Projects/Password%20Generator/)

### CODE REVIEW

**Comment by ankitapuri** [comment]
@jigar-sable can u please attach some screenshots

**Comment by arpit456jain** [comment]
@jigar-sable add it to json file also

**Comment by jigar-sable** [comment]
> @jigar-sable can u please attach some screenshots

image
Deployed link is also there.

**Review by ankitapuri — APPROVED** [review]

File: style_links.json [hunk 1]
```diff
@@ -71,5 +71,13 @@
...
     "name": "Tic-Tac-Toe 2",
     "link": "Tic Tac Toe/index.html"
+  },
+  {
+    "name": "Issue Tracker",
+    "link": "Issue Tracker"
+  },
+  {
+    "name": "Password Generator",
+    "link": "Password Generator"
```
Comment (ankitapuri) [inline style_links.json:81]: add link to html file

**Review by ankitapuri — CHANGES_REQUESTED** [review]

File: style_links.json (hunk 1 above)
Comment (jigar-sable) [reply style_links.json:81]: It will work. No need to add **/index.html**

**Review by ankitapuri — APPROVED** [review]

**Comment by arpit456jain** [comment]
@jigar-sable  please make PR from separete branch from next time not from master

**Comment by jigar-sable** [comment]
> @jigar-sable please make PR from separete branch from next time not from master

Alright

**Pull request merged**